gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/persistence/BackupManager.java

```diff
@@ -177,16 +197,22 @@ private File checkBaseline(File baselineParentDir) throws IOException{
       BackupInspector inspector = (baselineDir == null ? null : BackupInspector.createInspector(baselineDir));
             
       File storesDir = new File(backupDir, DATA_STORES);
-      RestoreScript restoreScript = new RestoreScript();
+      RestoreScript restoreScript =
+          this.incompleteRestoreScript == null ? new RestoreScript() : this.incompleteRestoreScript;
       HashSet<PersistentID> persistentIds = new HashSet<PersistentID>();
-      Collection<DiskStoreImpl> diskStores = new ArrayList<DiskStoreImpl>(cache.listDiskStoresIncludingRegionOwned());
+      // Collection<DiskStoreImpl> diskStores = new ArrayList<DiskStoreImpl>(cache.listDiskStoresIncludingRegionOwned());
+      Collection<DiskStoreImpl> diskStores = getDiskStoresToBackUp(diskStoresToBackup);
```

Comment:
May be I am missing something, but where is the order of disk stores specified if DISKSTORE_ALL is specified

Reply:
DISKSTORE_ALL etc cannot be specified the user at this point of time. In fact it will never be used in the current implementation. We can also remove it completely and always have the two phase order. Will remove it eventually if all works well in the testing.

Reply:
In fact as noted in the above comment, since I was thinking on the same lines that is why I continued with the DISKSTORE_ALL option, even though right now it is not used that later we can just connect the "ALL" option of the tool to this mode.